spec/fixtures/organizations.rb

```diff
@@ -1,18 +1,25 @@
 # frozen_string_literal: true
 
+# To please rubocop.
+def neu(inst, country, weight, sym, status = 1)
```

Comment:
This ends up leaking into the global namespace, which we should probably try to avoid, but you can use nested functions:

```ruby
def mock_organizations
  def neu(...)
    ...
  end
end
```

(I'd probably be inclined to call it `org` instead of `neu`, but that's just personal taste.)